formats/detailed_guide/frontend/examples/detailed_guide.json

```diff
@@ -0,0 +1,71 @@
+    "change_note": "The second entry in the table Examples of salary sacrifice has been amended to correct the explanation of how much of the salary is subject to tax and National Insurance contributions.",
+    "first_published_at": "2014-06-12T10:00:00+01:00"
+  },
+  "links": {
+    "lead_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
+        "base_path": "/government/organisations/hm-revenue-customs",
+        "api_url": "https://www.gov.uk/api/organisations/hm-revenue-customs",
+        "web_url": "https://www.gov.uk/government/organisations/hm-revenue-customs",
+        "locale": "en",
+        "analytics_identifier": "D25"
+      }
+    ],
+    "worldwide_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
```

Comment:
This isn't an accurate worldwide organisation. It leads to the example rendering in the frontend like this:
![screen shot 2016-04-05 at 17 46 38](https://cloud.githubusercontent.com/assets/319055/14290293/60d64da6-fb56-11e5-8983-64a97f1a8af0.png)
![screen shot 2016-04-05 at 17 46 35](https://cloud.githubusercontent.com/assets/319055/14290292/60d65e7c-fb56-11e5-8988-5c5df2a6f59c.png)